babel.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  presets: ['next/babel'],
+  plugins: [
+    [
+      'transform-imports',
+      {
+        '@material-ui/core/styles': {
+          // eslint-disable-next-line no-template-curly-in-string
+          transform: '@material-ui/styles/${member}',
+          // Whether or not to throw when an import is encountered which would cause the entire module to be imported.
+          preventFullImport: true
+        },
+        '@material-ui/core(?!/)': {
+          transform: importName =>
+            importName.toLowerCase().includes('style')
+              ? `@material-ui/styles/${importName}`
```

Comment:
This Internet is a struggle today... I had one question/comment that didn't come through:

Why are we installing and using `@material-ui/styles` directly? [Material UI docs](https://material-ui.com/guides/minimizing-bundle-size/#option-2) already describe how to handle if we're getting any errors when transforming `@material-ui/core/styles`

Reply:
@kilemensi things like `ServerStyleSheets` are not resolved `@material-ui/core/styles/ServerStyleSheets`